_partials/_multi-node-deprecation.md

```diff
@@ -0,0 +1,7 @@
+<Highlight type="warning">
+Multi-node support has been deprecated. TimescaleDB 2.13 is the last version that includes multi-node support. Multi-node support in 2.13 is available for PostgreSQL 13, 14 and 15.

```

Comment:
Still a lot of "multi-node" but I think this could be easier to understand:

```suggestion
Multi-node support has been deprecated with TimescaleDB 2.13. This is the last version that includes multi-node support and it supports multi-node installations with PostgreSQL versions 13, 14, and 15.
```